src/Standards/BestIt/Sniffs/Commenting/RequiredDocBlockSniff.php

```diff
@@ -87,11 +92,19 @@ private function checkAndRegisterMissingDocBlock(): void
         if (!$this->getDocCommentPos()) {
             $tokenIdent = $this->getTokenName();
 
-            $exception = (new CodeError(
-                static::CODE_MISSING_DOC_BLOCK_PREFIX . ucfirst($tokenIdent),
-                self::MESSAGE_MISSING_DOC_BLOCK,
-                $this->stackPos
-            ))->setPayload([lcfirst($tokenIdent)]);
+            if ($tokenIdent == $this->registeredTokens[T_VARIABLE]) {
+                $exception = (new CodeError(
```

Comment:
Please provide a unit test for this new case.

Reply:
Unittests with fixtures provided